optimizer/lib/TreeParser.js

```diff
@@ -113,6 +113,10 @@ Node.hasAttribute = function(attribute) {
   return attribute in this.attribs;
 };
 
+Node.insertText = function(text) {
+  htmlparser2.insertText(this, text);
```

Comment:
Should this return something?

Reply:
The original [insertText](https://github.com/ampproject/amp-toolbox/pull/70) doesn't return anything.